cc3200tool/cc.py

```diff
@@ -568,8 +610,12 @@ def format_slfs(self, size=None):
         size = SLFS_SIZE_MAP[size]
 
         log.info("Formatting flash with size=%s", size)
-        command = OPCODE_FORMAT_FLASH \
-            + struct.pack(">IIIII", 2, size/4, 0, 0, 2)
+        if PY3:
+            command = OPCODE_FORMAT_FLASH \
+                + struct.pack(">IIIII", 2, size//4, 0, 0, 2)
```

Comment:
Yes, I think `//` works for both Python 2 and 3. I'll remove the `if PY3:` test here.